index.bs

```diff
@@ -6554,6 +6577,98 @@ requests will be affected.
 
 </div>
 
+#### The network.setCacheBypass Command ####  {#command-network-setCacheBypass}
+
+The <dfn export for=commands>network.setCacheBypass</dfn> command bypasses the
+network cache for certain requests.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl remote-cddl">
+      network.SetCacheBypass = (
+        method: "network.setCacheBypass",
+        params: network.SetCacheBypassParameters
+      )
+
+      network.SetCacheBypassParameters = {
+        bypass: bool,
+        ? contexts: [+browsingContext.BrowsingContext]
+      }
+    </pre>
+   </dd>
+   <dt>Return Type</dt>
+   <dd>
+    <pre class="cddl">
+      EmptyResult
+    </pre>
+   </dd>
+</dl>
+
+<div algorithm>
+The <dfn export>WebDriver BiDi bypass cache</dfn> steps given |request| are:
+
+1. Let |context| be null.
+
+1. If |request|'s [=request/window=] is an [=environment settings object=]:
+
+  1. Let |environment settings| be |request|'s [=request/window=]
+
+  1. If there is a [=/browsing context=] whose [=active window=] is |environment
+     settings|' [=environment settings object/global object=], set |context| to
+     the [=top-level browsing context=] for that browsing context.
+
+1. If |context| is not null and [=navigable cache bypass set=] [=set/contains=]
+   |context|, return true.
+
+1.  Return [=default cache bypass=].
+
+</div>
+
+<div algorithm="remote end steps for network.setCacheBypass">
+The [=remote end steps=] given <var ignore>session</var> and |command parameters| are:
+
+1. Let |bypass| be |command parameters|["<code>bypass</code>"].
+
+1. If |command parameters| does not [=map/contain=] "<code>contexts</code>":
+
+ 1. Set the [=default cache bypass=] to |bypass|.
```

Comment:
```suggestion
  1. Set the [=default cache bypass=] to |bypass|.
```
I think here and another to points in this list, there should be one more space in the beginning, at the moment it's not align correctly in the generated html